pkg/addons/adminconsole/adminconsole.go

```diff
@@ -172,6 +173,10 @@ func (a *AdminConsole) Outro(ctx context.Context, cli client.Client, k0sCfg *k0s
 		return fmt.Errorf("unable to create kots password secret: %w", err)
 	}
 
+	if err := createKotsCAConfigmap(ctx, cli, a.namespace, a.privateCAs); err != nil {
```

Comment:
what happens if kotsadm gets deployed by helm before this configmap is created? would the kotsadm pod detect the configmap when it gets created?